libfwupdplugin/fu-bluez-device.c

```diff
@@ -340,7 +399,7 @@ fu_bluez_device_read (FuBluezDevice *self, const gchar *uuid, GError **error)
 /**
  * fu_bluez_device_read_string:
  * @self: A #FuBluezDevice
- * @uuid: The UUID, e.g. `00cde35c-7062-11eb-9439-0242ac130002`
+ * @uuid_str: The UUID, e.g. `00cde35c-7062-11eb-9439-0242ac130002`
```

Comment:
I kinda think the name should remain `uuid` -- and maybe call the Uuid object a `FuBluezDeviceHelper *helper` or `* uuid_obj` instead.